.gitignore

```diff
@@ -22,6 +22,7 @@
 .DS_Store
 .vscode
 /bin
+/metadata/generated
```

Comment:
`/metadata/generated` should go alphabetically under the header `# Generated by Yarn`